app/models/grda_warehouse/warehouse_reports/report_definition.rb

```diff
@@ -905,6 +905,15 @@ def self.report_list
           health: false,
         }
       end
+      if RailsDrivers.loaded.include?(:data_source_report)
+        r_list['Operational'] << {
+          url: 'data_source_report/warehouse_reports/reports',
+          name: 'Data Source Report',
+          description: 'Details of Data Sources',
```

Comment:
How about:
`description: 'Status and details of HMIS source data.`

Reply:
Yes.